homeassistant/components/skybell/config_flow.py

```diff
@@ -24,6 +25,10 @@ async def async_step_import(self, user_input: ConfigType) -> FlowResult:
             return self.async_abort(reason="already_configured")
         return await self.async_step_user(user_input)
 
+    async def async_step_reauth(self, user_input: Mapping[str, Any]) -> FlowResult:
+        """Handle a reauthorization flow request."""
+        return await self.async_step_user()
```

Comment:
We don't want them to be able to change the unique id / account in reauth. We should only ask them for the new password.